docs/lib/storage/fragments/native_common/getting-started/common.md

```diff
@@ -67,7 +67,7 @@ To upload to S3 from a data object, specify the key and the data object to be up
 
 Upon successfully executing this code, you should see an file called `myKey` in a folder called `public` with the data `My Data` in your S3 bucket.
 
-## Next Steps
+
```

Comment:
is this supposed to be removed from native Storage's getting started?

Reply:
Right, this one is removed last Friday I think. I forgot about this. I have put it back. But this would be fixed in another PR, I think